### PR TITLE
Fix Edge Loading

### PR DESCRIPTION
**Pull request by saucam**
This PR adds another fix to the data bug that was encountered previously, as we need to commit the transaction between adding the edge and its reverse edge

### CODE REVIEW

File: docs/Benchmarks.md [hunk 1]
```diff
@@ -93,9 +93,9 @@ We now commit the transaction after adding every edge/vertex in JanusGraph
 
 Run details
 
-JobId in history server: d0b4e985-6627-438f-960d-d44284c2b70f-0372
+JobId in history server: d0b4e985-6627-438f-960d-d44284c2b70f-0389
 
 | # of vertices | # of edges |  Vertex load time  | Edge load time | Total Job time | # of tasks failed |
 |---------------|------------|--------------------|----------------|----------------|-------------------|
-|477314|980989|59s s|1.4 min|6 min|0|
+|477314|981364|1 min|2.3 min|7.8 min|0|
```
Comment (JulienPeloton) [inline docs/Benchmarks.md:100]: Interesting - between the previous run and this one, there were only 0.04% more edge loaded (from 980989 to 981364) but the edge load time is almost doubled. Any ideas?

Comment (JulienPeloton) [reply docs/Benchmarks.md:100]: Or this is because it has to perform a full scan anyway for reverse edge (even if there are a few)?

Comment (saucam) [reply docs/Benchmarks.md:100]: Actually it has to do a commit of the transaction between the two edge additions now. Previously, it was only committing at the end, for both the 2 additions, so this is an increase in time per edge creation and hence a jump in time taken.
On the other hand, the API itself is not very reliable in committing the transactions, so it seems it was working most of the times but only skipping adding the edges on a few occasions (which could be due to multiple commits happening in parallel due to spark)

Comment (saucam) [reply docs/Benchmarks.md:100]: So basically the number of missed edges is random, and depends on each job run

Comment (JulienPeloton) [reply docs/Benchmarks.md:100]: OK, that makes sense - thanks!

**Review by JulienPeloton — APPROVED** [review]
Thanks for spotting this bug! Merging.

**Pull request merged**